File: seava.mod.bd.presenter/src/main/java/seava/bd/presenter/impl/currency/model/CurrencyLov_Ds.java
/**
 * Solutii Ecommerce, Automatizare, Validare si Analiza | Seava.ro
 * Copyright: 2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package seava.bd.presenter.impl.currency.model;

import seava.bd.domain.impl.currency.Currency;
import seava.j4e.api.annotation.Ds;
import seava.j4e.api.annotation.SortField;
import seava.j4e.presenter.impl.model.AbstractTypeWithCodeLov_Ds;

@Ds(entity = Currency.class, sort = {@SortField(field = CurrencyLov_Ds.f_code)})
public class CurrencyLov_Ds extends AbstractTypeWithCodeLov_Ds<Currency> {

	public static final String ALIAS = "bd_CurrencyLov_Ds";

	public CurrencyLov_Ds() {
		super();
	}

	public CurrencyLov_Ds(Currency e) {
		super(e);
	}
}

File: seava.mod.bd.ui.extjs/src/main/resources/webapp/seava/bd/ui/extjs/ds/CurrencyLov_Ds.js
/**
 * Solutii Ecommerce, Automatizare, Validare si Analiza | Seava.ro
 * Copyright: 2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
	
Ext.define("seava.bd.ui.extjs.ds.CurrencyLov_Ds", {
	extend: 'Ext.data.Model',
	
	statics: {
		ALIAS: "bd_CurrencyLov_Ds"
	},
	
	
	fields: [
		{name:"id", type:"string"},
		{name:"clientId", type:"string"},
		{name:"code", type:"string"},
		{name:"name", type:"string"},
		{name:"description", type:"string"},
		{name:"active", type:"boolean"},
		{name:"refid", type:"string"}
	]
});
